py-pkg/pkgnet/package_report.py

```diff
@@ -0,0 +1,104 @@
+import webbrowser
+from pathlib import Path
```

Comment:
what benefit are we getting from `pathlib` that we wouldn't from `os.path.abspath()`?

Reply:
In general, I'm just in favor of using `pathlib` instead of `os.path`. Both are standard library, but `pathlib.Path` just has a much nicer and easier-to-read interface.

Reply:
TIL it's standard lib, I didn't know! carry on...